### DISCUSSION TITLE
irs aided uav communication

### DISCUSSION THREAD

**Discussion by hdm29** [NVlabs/sionna#97] [Q&A] [answered]
how can i use Sionna for my thesis work on irs aided uav communication system? where can i get the data sets?

**Comment by jhoydis** [comment, marked as answer]
Hi,

In short, you currently can't. Sionna does not have a channel model for IRS or UAV. It also does not provide any datasets for this purpose.

**Reply by hdm29** [reply]
thank you for your fast response. I watched the video of those guys on wireless future podcast that i come to know about Sionna. So can i use it for classical channel models just for my discussion part?

**Reply by jhoydis** [reply]
The list of supported channel models can be found here: https://nvlabs.github.io/sionna/api/channel.html

**Reply by hdm29** [reply]
I meant from machine learning point of view. If i can use it for channel estimation.

**Reply by hdm29** [reply]
oh i see. let me check that first.
